external/test-runtime/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard;
```

Comment:
I would expect the second configuration to be netcoreapp not netstandard. I know you didn't set that and it was already that way but I think we should switch it to netcoreapp now that we are supporting multiple configurations.